packages/in_app_purchase/in_app_purchase_storekit/darwin/Classes/InAppPurchasePlugin.m

```diff
@@ -265,35 +248,27 @@ - (void)finishTransaction:(FlutterMethodCall *)call result:(FlutterResult)result
       @try {
         [self.paymentQueueHandler finishTransaction:transaction];
       } @catch (NSException *e) {
-        result([FlutterError errorWithCode:@"storekit_finish_transaction_exception"
-                                   message:e.name
-                                   details:e.description]);
+        *error = [FlutterError errorWithCode:@"storekit_finish_transaction_exception"
+                                     message:e.name
+                                     details:e.description];
```

Comment:
where is this error used?

Reply:
all generated pigeon functions have like an extra "error" param, I think it gets passed back to the dart side automatically